doit/runner.py

```diff
@@ -411,8 +412,15 @@ def _run_start_processes(self, job_q, result_q):
         # ### END DEBUG
 
         proc_list = []
-        for _ in range(self.num_process):
+        for i in range(self.num_process):
             next_job = self.get_next_job(None)
+            try:
+                delay = next_job.task_dict.get('delay') or 0
```

Comment:
in which occasion the attribute would not be present. tests?

anyway a `get('delay', 0)` would make the try/catch block unnecessary.